images/build/benv.sh

```diff
@@ -24,33 +24,18 @@ matchesName() {
 while read benvvar; do
   set -- "$benvvar" "$@"
 done < <(set | grep -i '^php=')
-while read benvvar; do
-  set -- "$benvvar" "$@"
-done < <(set | grep -i '^php_version=')
 while read benvvar; do
   set -- "$benvvar" "$@"
 done < <(set | grep -i '^python=')
-while read benvvar; do
-  set -- "$benvvar" "$@"
-done < <(set | grep -i '^python_version=')
 while read benvvar; do
   set -- "$benvvar" "$@"
 done < <(set | grep -i '^node=')
-while read benvvar; do
-  set -- "$benvvar" "$@"
-done < <(set | grep -i '^node_version=')
```

Comment:
2 things: 
1. I want to remove `benv` from supporting this since we have the logic (the spec which i sent in an email last week) within the c# code.
2. Sure we can consider both `node_vesion` and as well as `nodejs_version` in our code.